cpp/src/Ice/CollocatedRequestHandler.cpp

```diff
@@ -160,7 +160,7 @@ CollocatedRequestHandler::invokeAsyncRequest(OutgoingAsyncBase* outAsync, int ba
 
     if (!synchronous || !_response || _reference->getInvocationTimeout() > 0ms)
     {
-        auto stream = make_shared<InputStream>();
```

Comment:
One of the few spots we created a communicator-less InputStream for no reason since a communicator/instance is readily available.